mgmtsystem_action/models/mgmtsystem_action.py

```diff
@@ -1,15 +1,29 @@
 # -*- coding: utf-8 -*-
+# Copyright (C) 2010 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from openerp import fields, models, api, exceptions, _
+from odoo import models, api, fields, exceptions, _
 from datetime import datetime, timedelta
 
 
-class MgmtSystemAction(models.Model):
+class MgmtsystemAction(models.Model):
```

Comment:
@grindtildeath 
Did you try it out? The modules names are seperated by . 

So in class MgmtsystemAction
field _name is written as name = "mgmtsystem.action"
the casecamel has to bee accordingly or it throwse an error